src/lib.rs

```diff
@@ -12,12 +12,12 @@ use hdk::prelude::*;
 pub mod defs;
 pub use defs::*;
 
-const ROOT_ANCHOR: &'static str = concat!("holochain_anchors", "::", "root_anchor");
+pub const ROOT_ANCHOR: &'static str = concat!("holochain_anchors", "::", "root_anchor");
 pub const ANCHOR_TYPE: &'static str = concat!("holochain_anchors", "::", "Anchor");
-const ANCHOR_LINK_TYPE: &'static str = concat!("holochain_anchors", "::", "anchor_link");
+pub const ANCHOR_LINK_TYPE: &'static str = concat!("holochain_anchors", "::", "anchor_link");
```

Comment:
Same issue as above

Reply:
Hum oh I understand now what this crate wants to do. I was linking at the app level the parent anchor { type: "roles", name: "all_roles" } with every anchor { type: "roles", name: "editor" }, thanks. But now if I want to get all anchors with type: "roles", I need a function in the crate to do that, and I don't see how to do it as it is right now.

Reply:
All roles would just be `{ type: "roles", name: "" }` which is the parent. So no need for another function. You can get all roles of that anchor without an internal function

Reply:
Right, but then I need to do `get_links(parent_anchor, ANCHOR_LINK_TYPE, ANY)`, no?

If I don't have access to ANCHOR_LINK_TYPE or there is not any function to retrieve that internally, I have to replicate the ANCHOR_LINK_TYPE string outside the crate. Or I am missing something?

Reply:
No you are not ment to us the anchor directly. Instead you link from an anchor.
So it's like this:
```
let anchor_address = holochain_anchors::create_anchor("my_anchor_type".into(), "my_anchor".into())?;
hdk::utils::get_links_and_load_type(&anchor_address, LinkMatch::Exactly("my_link_type"), LinkMatch::Any)
```
Where `my_link_type` is defined in your zome. So it's your custom link type relevant to your zome. Holochain anchors doesn't know about it.

But honestly I'm not sure if this makes sense. I know the names are really bad and this needs to be thought out better. Perhaps we should have a call and chat about it soon.

Reply:
Yes I see, the only thing missing for me is a `get_all_anchors_of_type(anchor_type: String)` function or something like it. If that makes sense I can fix this PR and you can merge it, otherwise we can make the call :)

Reply:
Im closing this one.